selinux/ceph-installer.fc

```diff
@@ -0,0 +1,11 @@
+/usr/bin/gunicorn_pecan		--	gen_context(system_u:object_r:ceph_installer_exec_t,s0)
```

Comment:
Yeah, I tried Lukas' policy and it was missing several permissions :(

I think this PR should include the wrapper scripts and label those.